Makefile

```diff
@@ -64,16 +64,19 @@ SHELL := /bin/bash
 
 
 default:
-	cd key/aziot-keys/ && $(CBINDGEN) --config cbindgen.toml --output aziot-keys.h.tmp $(CBINDGEN_VERBOSE)
-	cp key/aziot-keys/cbindgen.prelude.h key/aziot-keys/aziot-keys.h.new
-	< key/aziot-keys/aziot-keys.h.tmp grep -v 'cbindgen_unused_' >> key/aziot-keys/aziot-keys.h.new
-	$(RM) key/aziot-keys/aziot-keys.h.tmp
-	if ! diff -q key/aziot-keys/aziot-keys.h key/aziot-keys/aziot-keys.h.new; then \
-		mv key/aziot-keys/aziot-keys.h.new key/aziot-keys/aziot-keys.h; \
+	# Re-generate aziot-keys.h if necessary
+	set -euo pipefail; \
```

Comment:
any chance we could use something like https://www.gnu.org/software/make/manual/html_node/One-Shell.html (see [also](https://stackoverflow.com/questions/53541081/embedding-bash-script-into-makefile)) to avoid all the `\` at the end of each line?

Reply:
The non-oneshell uses outweigh the oneshell uses, so I'd rather not have to remember to prefix every target with `set -e`.

Reply:
Sure, that's fine. Just thought I'd bring it up.